### PR TITLE
[lcn] Fix setting of variables

### PR DESCRIPTION
**Pull request by fwolter**
Signed-off-by: Fabian Wolter <github@fabian-wolter.de>

### CODE REVIEW

**Review by jlaur — APPROVED** [review]
Thanks! I assume that zero-padding to three digits was not working. :smile:

**Comment by jlaur** [comment]
@fwolter - before merging, should we wait for user confirmation, or were you able to test this?

**Comment by fwolter** [comment]
Yeah, *seemed* to be a good idea the padding. The user and I tested it successfully.

**Pull request merged**
